### DISCUSSION TITLE
Add codesearch to iOS app

### DISCUSSION THREAD

**Discussion by 111andre111** [community/community#17465] [Mobile]
It would be great if codesearch feature could be added to that app
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.66.0 (68112067)
Device: iPhone 13 Pro Max (iOS 15.4.1)
</details>

**Comment by HenSquared** [comment]
Hi @111andre111, 

Thanks for reaching out and submitting feedback. I opened an issue tracking this request to discuss internally with the team. 

I am assuming you mean GitHub Code Search: https://cs.github.com/about

Please let me know if this is not the feature you are requesting. Thanks :)

**Comment by 111andre111** [comment]
Hi @HenSquared ,

yes absolutely I was talking about Github Code Search, exactly.

**Comment by 111andre111** [comment]
I am sorry, @HenSquared 
let me be more precise. I meant, that Github Code Search would be available from within the Github app iOS:
https://apps.apple.com/us/app/github/id1477376905
Sorry, that I've not been precise enough before.

**Reply by HenSquared** [reply]
Got it! I understand. Thanks for the additional clarification :)

**Comment by AbhinavKoul** [comment]
Is there an update on this feature?

**Reply by Iron-Ham** [reply]
Hi Abhinav, we don't have an official update to share at this time -- but this is something that's being actively worked on. 

We'll be able to provide a more detailed update in the future! 


Thanks,
Hesham

**Comment by elisealix22** [comment]
Hi @111andre111 and @AbhinavKoul , we recently published a new version of GitHub Mobile (v1.133 and above) that adds code search to repositories. Thanks for being patient with us and let us know if you have any other suggestions!
